htdocs/core/modules/modSociete.class.php

```diff
@@ -203,7 +203,7 @@ public function __construct($db)
 		$this->rights[$r][0] = 130;
 		$this->rights[$r][1] = 'Modify thirdparty information payment';
 		$this->rights[$r][3] = 0;
-		$this->rights[$r][4] = 'thirdparty_paymentinformation_advance';      // Visible if option MAIN_USE_ADVANCED_PERMS is on
+		$this->rights[$r][4] = 'thirdparty_paymentinformation';
```

Comment:
The idea behind the permission is that we must have in the default mode only 3 permission
read, update/write and delete
+ some if it is really not possible.

As soon as we want to have a more accurate permission level, it is advanced permissions.